02 - Community Expansions/🗂️ 02 - Community Expansions.md

```diff
@@ -19,7 +19,7 @@ If you prefer to browse the full directory of *all* expansions provided by the O
 %% Zoottelkeeper: Beginning of the autogenerated index file list  %%
 -  [[02 - Community Expansions/02.01 Plugins by Category/🗂️ 02.01 Plugins by Category|🗂️ 02.01 Plugins by Category]]
 -  [[02 - Community Expansions/02.02 Themes by Category/🗂️ 02.02 Themes by Category|🗂️ 02.02 Themes by Category]]
--  [[02 - Community Expansions/02.03 CSS Snippets by Category/🗂️ 02.03 CSS Snippets by Category|🗂️ 02.03 CSS Snippets by Category]]
--  [[02 - Community Expansions/02.04 Auxiliary Tools by Category/🗂️ 02.04 Auxiliary Tools by Category|🗂️ 02.04 Auxiliary Tools by Category]]
+-  [[02 - Community Expansions/02.04 Auxiliary Tools by Category/🗂️ 02.04 Auxiliary Tools by Category|🗂️ 02.03 CSS Snippets by Category]]
+-  [[🗂️ 02.03 CSS Snippets by Category|🗂️ 02.04 Auxiliary Tools by Category]]
```

Comment:
This will be fixed by regenerating the MOCs, but just to note that this now means that clicking on the text 02.03 takes you to 02.04 - and vice versa...

Reply:
yeah, I'll fix this in a separate PR once #84  is merged (so we can do it in one go)